pkg/apis/application/v1alpha1/applicationset_types.go

```diff
@@ -86,6 +86,7 @@ type ApplicationSetStrategy struct {
 	Type        string                         `json:"type,omitempty" protobuf:"bytes,1,opt,name=type"`
 	RollingSync *ApplicationSetRolloutStrategy `json:"rollingSync,omitempty" protobuf:"bytes,2,opt,name=rollingSync"`
 	// RollingUpdate *ApplicationSetRolloutStrategy `json:"rollingUpdate,omitempty" protobuf:"bytes,3,opt,name=rollingUpdate"`
+	DeletionOrder string `json:"deletionOrder,omitempty" protobuf:"bytes,3,opt,name=deletionOrder"`
```

Comment:
Add comment for this new field.

Reply:
done in [1d68c42](https://github.com/argoproj/argo-cd/pull/23465/commits/1d68c42f75ef828c7a2173ae3329d312740efd1f)